playwright-tests/properties-page-tests.spec.ts

```diff
@@ -0,0 +1,17 @@
+import { test, expect } from '@playwright/test';
+
```

Comment:
**Optional**: consider setting up a helper object for properties like I did for the users